components/match2/commons/starcraft_starcraft2/game_summary_starcraft_ffa.lua

```diff
@@ -0,0 +1,56 @@
+---
+-- @Liquipedia
+-- wiki=commons
+-- page=Module:GameSummary/Starcraft/FFa
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local CustomGameSummary = {}
+
+local Array = require('Module:Array')
+local Lua = require('Module:Lua')
+
+local MatchGroupUtil = Lua.import('Module:MatchGroup/Util/Starcraft')
+
+local SummaryHelper = Lua.import('Module:MatchSummary/Base/Ffa')
+local MatchSummaryWidgets = Lua.import('Module:Widget/Match/Summary/Ffa/All')
+
+---@param props {bracketId: string, matchId: string, gameIdx: integer}
+---@return Html
+function CustomGameSummary.getGameByMatchId(props)
+	local match = MatchGroupUtil.fetchMatchForBracketDisplay(props.bracketId, props.matchId)
+
+	local game = match.games[props.gameIdx]
+	assert(game, 'Error Game ID ' .. tostring(props.gameIdx) .. ' not found')
+
+	game.stream = match.stream
+
+	SummaryHelper.updateGameOpponents(game, match.opponents)
+
+	return MatchSummaryWidgets.Tab{
+		matchId = match.matchId,
+		idx = props.gameIdx,
+		children = {
+			MatchSummaryWidgets.GameDetails{game = game},
+			SummaryHelper.standardGame(game, CustomGameSummary)
+		}
+	}
+end
+
+---@param columns table[]
+---@param game table
+---@return table[]
+function CustomGameSummary.adjustGameStandingsColumns(columns, game)
+	return Array.map(columns, function(column)
+		if column.id == 'totalPoints' and game.extradata.settings.noscore then
+			return
+		elseif column.id == 'placements' or column.id == 'kills' then
+			return
+		end
```

Comment:
This case can be handled by setting `showgamedetails` in settngs in MGI to false